docs/manual/install/index.rst

```diff
@@ -79,7 +79,7 @@ cairocffi
 
 Qtile uses cairocffi_ for drawing on status bars and popup windows. Under X11,
 cairocffi requires XCB support via xcffib, which you should be sure to have
-installed **before** installing cairocffi, otherwise the needed cairo-xcb
+installed **before** installing cairocffi, otherwise, the needed cairo-xcb
```

Comment:
I think the comma after "otherwise" is fine. However, I think it should be a semi-colon before it, rather than a comma.